artemis/config_templates/brokered/broker.xml

```diff
@@ -96,8 +96,10 @@ under the License.
       <global-max-size>104857600</global-max-size>
 
       <acceptors>
-         <acceptor name="amqp">tcp://0.0.0.0:5672?protocols=AMQP</acceptor>
-         <acceptor name="amqps-internal">tcp://0.0.0.0:5673?protocols=AMQP;sslEnabled=true;keyStorePath=${KEYSTORE_PATH};keyStorePassword=enmasse;trustStorePath=${TRUSTSTORE_PATH};trustStorePassword=enmasse;verifyHost=false;needClientAuth=true</acceptor>
+         <acceptor name="amqp">tcp://0.0.0.0:5672?protocols=AMQP;saslMechanisms=PLAIN</acceptor>
+         <acceptor name="amqps">tcp://0.0.0.0:5671?protocols=AMQP;sslEnabled=true;saslMechanisms=PLAIN;keyStorePath=${KEYSTORE_PATH};keyStorePassword=enmasse</acceptor>
+         <!-- SASL Mechanism should be EXTERNAL not anonymous... but Artemis doesn't support EXTERNAL -->
+         <acceptor name="amqps-internal">tcp://0.0.0.0:55671?protocols=AMQP;saslMechanisms=ANONYMOUS;sslEnabled=true;keyStorePath=${KEYSTORE_PATH};keyStorePassword=enmasse;trustStorePath=${TRUSTSTORE_PATH};trustStorePassword=enmasse;verifyHost=false;needClientAuth=true</acceptor>
```

Comment:
Should be amqps-normal

Reply:
I assume the names don't actually matter here, but I shall change it presently